apps/webpack.js

```diff
@@ -20,7 +21,9 @@ var baseConfig = {
     "blockly": "this Blockly",
     "react": "var React",
     "react-dom": "var ReactDOM",
-    "jquery": "var $"
+    "jquery": "var $",
+    "radium": "var Radium",
+    "bindings": true
```

Comment:
My goal was to start by having everything be as much the same as before as possible (except built by webpack instead of browserify). i.e. my goal was to initial produce bundles that are pretty much exactly the same as before.

I think this is a good example of a next step that we can take now that they are combined, which will involve making sure we include the right outputs everywhere in dashboard.